docs/sign-up.rst

```diff
@@ -0,0 +1,71 @@
+
+.. _sign-up:
+
+=======
+Sign up
+=======
+
+This page shows you how to create a CrateDB Cloud user account.
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+
+.. _create-account:
+
+Create an account
+=================
+
+To create an account, call up the `CrateDB Cloud sign-up page`_ and complete the
+form.
```

Comment:
then say complete the form, etc, then say, "when you're done, it should look like this:" or whatever